tests/tests_integration/test_api/test_files.py

```diff
@@ -20,6 +21,23 @@ def new_file():
     assert COGNITE_CLIENT.files.retrieve(id=res.id) is None
 
 
+@pytest.fixture(scope="class")
+def new_file_with_label():
+    label_external_id = uuid.uuid4().hex[0:20]
+    COGNITE_CLIENT.labels.create(LabelDefinition(external_id=label_external_id, name="mandatory"))
```

Comment:
```suggestion
    random_label = COGNITE_CLIENT.labels.create(LabelDefinition(external_id=label_external_id, name="mandatory"))
```

and then on line 35, 
```
 yield file, random_label.external_id
```

This way you would know that the label indeed got created in the first place.